src/unit_test.rs

```diff
@@ -39,10 +41,20 @@ fn build_tests(path: &PathBuf) -> Result<Vec<UnitTest>, Vec<String>> {
     crate::topology::unit_test::build_unit_tests(&config)
 }
 
+lazy_static! {
+    static ref DEFAULT_PATHS: Vec<PathBuf> = vec!["/etc/vector/vector.toml".into()];
```

Comment:
I think we have this in three places now, so it might be good to condense that down to one. With Rust 1.39.0 I don't think we even need `lazy_static` anymore, since `Vec::new` is a const fn.